src/package/polyfill/createHookClass.ts

```diff
@@ -2,18 +2,50 @@ import { getMostRecentImpl } from './hmrCache';
 
 export function createHookClass(elementName: string, originalImpl: any) {
     return class extends originalImpl {
-        static get observedAttributes() {
-            return super.observedAttributes;
-        }
-
         connectedCallback() {
             const mostRecentImpl = getMostRecentImpl(elementName).prototype;
+
+            const ctx = this;
+            const observerOptions = {
+                childList: false,
+                attributes: true,
+                subtree: false
+            };
+
+            function callback(mutationList: any[], observer: MutationObserver) {
```

Comment:
It has no `this` scope, so it could be a fat arrow function :) `const callback = (...) => { ... }` Minimal performance gain but well :)

Reply:
True, copy paste... Havent used this observer before 😂

Reply:
fixed